### DISCUSSION TITLE
Is it possible to get notified before validation on submit?

### DISCUSSION THREAD

**Discussion by nanandn** [react-hook-form/react-hook-form#10579] [General]
Is it possible to get notified before validation on submit? I want to show a "busy" state when the form is submitted. "handleSubmit" is called after the validation is successful. 

Thanks
Anand

**Comment by leapful** [comment]
@nanandn 

You can use `isValidating` or `isSubmitting` for this case.

Reference: https://react-hook-form.com/docs/useform/formstate
